controls/package_spec.rb

```diff
@@ -101,3 +101,74 @@
     its('disk_error_action') { should cmp 'SUSPEND' }
   end
 end
+
+
+control 'package-09' do
+  impact 1.0
+  title "Install packages suggested by Lynis"
```

Comment:
Mmmm...I'm somehow not really happy with that, by this change we force to install all this packages on each system. There are things like `fail2ban`, `arpwatch` etc. If I take some admin perspective, I probably would not really like to be forced to have this packages, but I would rather expect to have it like opt-in and to decide it by myself. This things usually need tuning and configuration and their requirement & usage depends on the particular environment.

Basically from my view its not generic enough in order to have it here

@bitvijays can we maybe avoid this completely? Do you see some other options?

@rndmh3ro @atomic111 whats your view on this?